experiments/interpretation/dutchf3_patch/local/azureml_requirements.txt

```diff
@@ -0,0 +1,13 @@
+git+https://github.com/olgaliak/seismic-deeplearning.git@azureml-train-pipeline#egg=cv_lib&subdirectory=cv_lib
+git+https://github.com/microsoft/seismic-deeplearning.git#egg=deepseismic-interpretation&subdirectory=interpretation
+opencv-python==4.1.2.30
```

Comment:
Is it possible to use Pillow for your work instead of openCV?

Reply:
I think we are only using OpenCV for the boarder_contant feature to pad images. This is in the train.py that we are leveraging and in other areas of the code. Are you recommending to substitute this for similar functionality in Pillow?